src/browser.ts

```diff
@@ -57,12 +55,16 @@ class Honeybadger extends Client {
     return (this.__errorsSent = 0)
   }
 
-  factory(opts?: Partial<BrowserConfig>): Honeybadger {
+  public factory(opts?: Partial<BrowserConfig>): Honeybadger {
     return new Honeybadger(opts)
   }
 
+  public checkIn(_id: string): Promise<void> {
```

Comment:
This is only here because of Typescript. Our TS definitions will show that `checkIn` is available (because we export `Server & Browser` in a single object. So I added this to throw an more specific error instead of an error like _Uncaught TypeError: Honeybadger.checkIn is not a function_